src/puppetlabs/trapperkeeper/services/status/status_proxy_service.clj

```diff
@@ -0,0 +1,22 @@
+(ns puppetlabs.trapperkeeper.services.status.status-proxy-service
+  (:require [clojure.tools.logging :as log]
+    [puppetlabs.trapperkeeper.core :refer [defservice]])
+  (:import (java.net URL)))
+
+(defservice status-proxy-service
+  [[:WebroutingService add-proxy-route]
+   [:ConfigService get-in-config]]
+  (init [this context]
+    (log/info "Initializing status service proxy")
+    (let [target-url (URL. (get-in-config [:status-proxy :proxy-target-url]))
+          host (.getHost target-url)
+          port (.getPort target-url)
+          path (.getPath target-url)]
+      (add-proxy-route
+        this
+        {:host host
+         :port port
+         :path path}
+        {:ssl-config (get-in-config [:status-proxy :ssl-opts])
```

Comment:
If `ssl-opts` is not supplied in the configuration, what does this do?  Will `add-proxy-route` effectively use `:use-server-config` (which is probably a reasonable thing to do) or will it blow up?  Looks like from the schema validation that tk-jetty9 does that it might blow up - https://github.com/puppetlabs/trapperkeeper-webserver-jetty9/blob/1.3.1/src/puppetlabs/trapperkeeper/services/webserver/jetty9_core.clj#L82-L84.